### PR TITLE
add format validation for ci

### PR DESCRIPTION
**Pull request by mathieu-pousse**
Signed-off-by: Mathieu POUSSE <mathieu.pousse@zenika.com>

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -6,8 +6,10 @@ import (
 	"os"
 )
 
-func main() {
+func main() 
+{
```
Comment (ggerbaud) [inline main.go]: Compilation issue

**Comment by mathieu-pousse** [comment]
check it out les ploucs @ggerbaud @erwannT

**Comment by erwannT** [comment]
LGTM

**Comment by ggerbaud** [comment]
ouais pareil...

**Pull request merged**